tools/requirements.txt

```diff
@@ -1,12 +1,12 @@
-Logbook==1.5.3
-structlog==19.2.0
-taskcluster==24.0.1
-raven==6.10.0
 
 # limit async-timeout version to avoid using the alpha 4.0.0a0
 # still breaking the events build
 async-timeout<4.0,>=3.0
+Logbook==1.5.3
 
 # limits to avoid building with gcc on alpine
```

Comment:
this comment is not true for the list of requirements below

Reply:
I've removed it.